packages/reactive-element/src/css-tag.ts

```diff
@@ -161,7 +161,7 @@ export const css = (
 /**
  * Applies the given styles to a `shadowRoot`. When Shadow DOM is
  * available but `adoptedStyleSheets` is not, styles are appended to the
- * `shadowRoot` to [mimic spec behavior](https://wicg.github.io/construct-stylesheets/#using-constructed-stylesheets).
+ * `shadowRoot` to [mimic spec behavior](https://github.com/WICG/construct-stylesheets/blob/main/explainer.md#example-usage).
```

Comment:
I think an MDN link is even better, now that it's documented there.

```suggestion
 * `shadowRoot` to [mimic the native feature](https://developer.mozilla.org/en-US/docs/Web/API/ShadowRoot/adoptedStyleSheets).
```